scheduler/src/main/resources/application.properties

```diff
@@ -1,28 +1,26 @@
-#datasource for application and quartz
-
-spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://127.0.0.1/autoscaler
-spring.datasource.username=postgres
-spring.datasource.password=postgres
-
-#quartz job
-scalingenginejob.reschedule.interval.millisecond=10000
-scalingenginejob.reschedule.maxcount=6
-scalingengine.notification.reschedule.maxcount=3
+# datasource for application and quartz
```

Comment:
We kept application.properties for reference only. Just to know what properties are being used.
If you think it's not required, we can remove it?